doc/playbook.rst

```diff
@@ -249,7 +249,7 @@ One raw was ingested, visit-defined, and kept in the development central repo, s
 
 .. code-block:: sh
 
-   make_apdb.py -c db_url="sqlite:///apdb.db"
+   apdb-cli create-sql "sqlite:///apdb.db" apdb_config.py
    pipetask run -b s3://rubin-pp-dev-users/central_repo -i LATISS/raw/all,LATISS/defaults,LATISS/templates -o u/username/collection  -d "detector=0 and instrument='LATISS' and exposure=2023082900500 and visit_system=0" -p $PROMPT_PROCESSING_DIR/pipelines/LATISS/ApPipe.yaml -c diaPipe:apdb.db_url=sqlite:///apdb.db --register-dataset-types
```

Comment:
Hmm, this is a problem -- until DM-43416, `pipetask run` doesn't let us actually use `apdb_config.py`.

```suggestion
   pipetask run -b s3://rubin-pp-dev-users/central_repo -i LATISS/raw/all,LATISS/defaults,LATISS/templates -o u/username/collection  -d "detector=0 and instrument='LATISS' and exposure=2023082900500 and visit_system=0" -p $PROMPT_PROCESSING_DIR/pipelines/LATISS/ApPipe.yaml -c diaPipe:apdb.db_url=sqlite:///apdb.db --register-dataset-types

.. TODO: update pipetask call after DM-43416
```

Reply:
Well, it's not a problem if you just ignore that file for now and keep using the same pipetask options.